smart-web-server/src/main/java/org/smartdata/server/generated/api/SystemApiController.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.smartdata.server.generated.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.annotation.Generated;
+
+import java.util.Optional;
+
+@Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
+@RestController
+@RequestMapping("${openapi.sSMAPIDocumentation.base-path:}")
```

Comment:
may be here we should have smth like: ${openapi.sSMAPIDocumentation.base-path:} + /api/v2/system and in SystemApi interface on the method getCurrentUser() smth like this: /current-user ?

Reply:
Same as above